src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java

```diff
@@ -162,10 +162,10 @@
 			MessageBundle.getMessage("angal.common.name.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.common.age.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.common.sex.txt").toUpperCase(),
-			MessageBundle.getMessage("angal.admission.cityaddresstelephonenote.txt").toUpperCase(),
-			MessageBundle.getMessage("angal.admission.ward.txt").toUpperCase()
+			MessageBundle.getMessage("angal.admission.cityaddresstelephonenote.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.ward.txt").toUpperCase()
```

Comment:
in which case we should have .col or .txt?

Reply:
I am using ".col" only when it is a table column heading; if it is both used as a column heading and it other places like dialog labels, or just text then I use ".txt".